openpype/hosts/houdini/api/lib.py

```diff
@@ -22,9 +22,12 @@
 JSON_PREFIX = "JSON:::"
 
 
-def get_asset_fps():
+def get_asset_fps(asset_doc=None):
     """Return current asset fps."""
-    return get_current_project_asset()["data"].get("fps")
+
+    if asset_doc is None:
+        asset_doc = get_current_project_asset(fields=["data.fps"])
+    return asset_doc["data"].get("fps")
```

Comment:
I know this is the way "it was before" but the `.get("fps")` doesn't really make much sense looking at how `get_asset_fps` is used. It's used to set the FPS in `reset_frame_range` and that logic NEVER expects `None` to be the fps value.

As such, should we maybe just start 'believing' `fps` to be available on the asset so we can at least rely on this `get_asset_fps` function to return some data from the asset and error if otherwise?

```suggestion
    return asset_doc["data"]["fps"]
```